cafebabel/translations/models.py

```diff
@@ -0,0 +1,53 @@
+from flask import url_for
+from mongoengine import signals, ValidationError
+
+from .. import app, db
+from ..users.models import User
+from ..articles.models import Article
+
+
+class Translation(Article):
```

Comment:
`translations.models.Translation` sounds very generic while we are being as specific as the translation of an article.
Would it make sense to give a more specific name?

Same question for the indenpendant package. if is so related to the _articles_ package that I'm wondering why extracting it.

We will have translations soon for the system, maybe we don't want the names to be confusing. (Or should we merge the _translations_ module and make it the website translation module? In which case the `class Translation` would still be too specific) 🤔

Reply:
1. on translation specificity, I think we should keep it and name the translation of the interface internationalization (i18n) to make the difference
2. for the independence, I found the `articles` folder already too crowded, when you have more than one blueprint in views it's a sign _to me_, I even want to go deeper and extract drafts too with their own set of blueprint/views/templates

For instance here: https://github.com/cafebabel/cafebabel.com/blob/master/templates/articles/edit.html#L4-L13 To me again that rang a bell, sometimes it's less costly to repeat ourselves a bit vs. complexity of the logic (and probably more to come).

Reply:
I agree on point 2.
In order to keep items splitted and yet consistently connected, what do you think of making _drafts_, _translations_ (and current _articles_?) subpackages of the _articles_ package?
In which case we would import _Translation_ from _articles_ and that would still be readable to me ^^

If you'd rather this with 1st level packages I'd suggest just renaming the _Translation_ model subclass _ArticleTranslation_ for the sake of explicity.

Reply:
I'll go for moving the folder… which makes me thinking about URLs :)

What about `article/translation/…`? (and same for draft and proposal)

Reply:
That would definitely make sense to me indeed. 👍